boltdesignsystem/bolt
# DS-576 fix static page header dropdown position

## Jira

https://pegadigitalit.atlassian.net/browse/DS-576

## Summary

Fixes an issue with static page header's top level item dropdown does not move with the page header as the user scrolls.

## Details

1. Updated CSS to use absolute position rather than fixed
2. `flat` prop has been moved from ul template to li template

### How to make a top level item use a single level flat dropdown

```twig
{% set flat_lv2_ul_content %}
  {% include '@bolt-components-page-header/page-header-nav-li.twig' with {
    link: {
      content: 'Flat Level 2',
      attributes: {
        href: 'https://pega.com',
      },
    },
  } only %}
{% endset %}
{% set flat_lv2 %}
  {% include '@bolt-components-page-header/page-header-nav-ul.twig' with {
    content: flat_lv2_ul_content,
  } only %}
{% endset %}

{% include '@bolt-components-page-header/page-header-nav-li.twig' with [
  {
    link: {
      content: 'Why Pega?',
    },
    children: flat_lv2,
    flat: true,
  },
] only %}
```

## How to test

Run the branch locally and check a demo page that has a static page header, hover over each top level items to see the dropdowns. Scroll the page and make sure the dropdowns move with the page header.

http://localhost:3000/pattern-lab/?p=pages-d8-browse-page-media
